frontend/src/layout/navigation/TopBar/notificationsLogic.ts

```diff
@@ -31,16 +33,33 @@ export const notificationsLogic = kea<notificationsLogicType>([
                         const response = (await api.get(
                             `api/projects/${teamLogic.values.currentTeamId}/activity_log/important_changes`
                         )) as ActivityLogItem[]
+                        // we can't rely on automatic success action here because we swallow errors so always succeed
+                        actions.clearErrorCount()
                         return humanize(response, describerFor, true)
+                    } catch (e) {
+                        // swallow errors as this isn't user initiated
+                        // increment a counter to backoff calling the API while errors persist
+                        actions.incrementErrorCount()
+                        return []
                     } finally {
-                        const timeout = window.setTimeout(actions.loadImportantChanges, POLL_TIMEOUT)
+                        const pollTimeoutMilliseconds = values.errorCounter
```

Comment:
This could do with a `Math.max(SOMETHING_REASONABLE)`. It's a small detail but if we don't do it now we never will and one day far in the future it will be a weird hard to understand bug 😅